ansible/vars/production/production_public.yml

```diff
@@ -35,7 +35,6 @@ primary_ssl_env: "production"
 dev_group: 'dimagidev'
 dev_users:
   present:
-    - ansible
```

Comment:
Would be nice to move `dev_group` and `dev_users` into a single vars file so they don't have to be duplicated on each environment. I didn't look closely, but guessing they're all the same.